R/FilteredDataCDISC.R

```diff
@@ -310,9 +312,11 @@ CDISCFilteredData <- R6::R6Class( # nolint
 #' @keywords internal
 #'
 #' @examples
-#' teal.slice:::topological_sort(list(A = c(), B = c("A"), C = c("B"), D = c("A")))
-#' teal.slice:::topological_sort(list(D = c("A"), A = c(), B = c("A"), C = c("B")))
-#' teal.slice:::topological_sort(list(D = c("A"), B = c("A"), C = c("B"), A = c()))
+#' \dontrun{
```

Comment:
why not to run?

Reply:
In order to not use `:::` and avoid check errors.